portal/static/js/initialQueries.js

```diff
@@ -30,10 +8,11 @@
                 throw new Error("Dependency with key value: " + key + " not found.");
             }
         };
-        this.userId = userId;
+        this.userId = 0;
```

Comment:
I might use a non-numeric value (`null`?) if it hasn't been initialized to make it easier to distinguish...